test/src/main/java/org/apache/accumulo/test/functional/AccumuloClientIT.java

```diff
@@ -78,49 +96,53 @@ public void testAccumuloClientBuilder() throws Exception {
     AccumuloClient c = Accumulo.newClient().from(getClientProps()).build();
     String instanceName = getClientInfo().getInstanceName();
     String zookeepers = getClientInfo().getZooKeepers();
-    final String user = "testuser";
-    final String password = "testpassword";
-    c.securityOperations().createLocalUser(user, new PasswordToken(password));
 
-    AccumuloClient client = Accumulo.newClient().to(instanceName, zookeepers).as(user, password)
+    ClusterUser testuser1 = getUser(0);
+    final String user1 = testuser1.getPrincipal();
+    final String password1 = testuser1.getPassword();
+    c.securityOperations().createLocalUser(user1, new PasswordToken(password1));
```

Comment:
The purpose of the ClusterUser abstraction is to allow test to work with passwords or kerberose.  For this test to work with kerberose it should not use `PasswordToken()`, but instead use `testuser1.getToken()`

Reply:
Thanks.  I'll assume a kerberose check is needed.

Reply:
Hopefully `testuser1.getToken()` is an easy change and it works with password or kerberose.  If not this could be a follow on issue.

Reply:
Nevermind, this is not as simple as I thought.  I was thinking `c.securityOperations().createLocalUser(user1, testuser1.getToken())`.  However  I looked at the method `createLocalUser` and it explicitly takes a PasswordToken.  I saw some other test code passes null for the token when sasl is enabled, but I Am not sure what the correct thing to do.